doc/config.md

```diff
@@ -0,0 +1,118 @@
+# Configuration format of qem-bot
+
+Configuration, in another way also called metadata of qem-bot is held in yaml files with one file per "product". Additionally, there is one special config file called `singlearch.yml` which contains a list of packages that exist only on a single architecture.
```

Comment:
Since the configuration spans multiple files, maybe it would be a good idea to start this with a directory listing of these YAML files, before describing them individually in more detail. To get an overview of what's being described in the document.
```
|- singlearch.yaml
...
```